imdb_django/management/commands/import_principals.py

```diff
@@ -0,0 +1,78 @@
+# myapp/management/commands/import_principals_data.py
+
+import csv
+from django.core.management.base import BaseCommand
+from django.db import transaction
+from imdb_django.models import Name, Title, TitlePrincipal
+from helper import log_info  # Import the log_info function
+
+# Initialize the logger
```

Comment:
I guess we move the logger but comment is still here

Please remove this comment in all files where needed